client/my-sites/stats/activity-log-banner/progress-banner.jsx

```diff
@@ -11,31 +11,44 @@ import ActivityLogBanner from './index';
 import ProgressBar from 'components/progress-bar';
 
 function SuccessBanner( {
-	translate,
 	moment,
+	percent,
+	status,
+	timestamp,
+	translate,
 } ) {
-	// FIXME: real dates
-	const date = 1496400468285;
-	// FIXME: real progress
-	const progress = 25;
-
 	return (
 		<ActivityLogBanner
 			status="info"
 			title={ translate( 'Currently restoring your site' ) }
 		>
 			<p>{ translate(
-				'We\'re in the process of restoring your site back to %s.' +
-				'You\'ll be notified once it\'s complete.',
-				{ args: moment( date ).format( 'LLLL' ) }
+				"We're in the process of restoring your site back to %s." +
+				"You'll be notified once it's complete.",
+				{ args: moment( timestamp ).format( 'LLLL' ) }
 			) }</p>
 
 			<div>
-				<em>{ translate( 'Currently restoring posts…' ) }</em>
-				<ProgressBar value={ progress } isPulsing />
+				<em>{
+					/*
+					* FIXME: Do we have a detailed message or should this be removed?
+					* FIXME: Show a message for `queued` status before progress?
```

Comment:
Good question. This is very obvious at the moment because everything gets queued forever. Maybe we'll want to make the bar pulse and show a "Queued..." message. Can be done in a separate PR though.